packages/cli/src/collect/collect.js

```diff
@@ -167,7 +173,11 @@ async function startServerAndDetermineUrls(options) {
 
   const urls = urlsAsArray;
   if (!urls.length) {
-    urls.push(...server.getAvailableUrls().slice(0, 5));
+    if (typeof options.numberOfFiles !== 'undefined' && options.numberOfFiles <= 0) {
+      urls.push(...server.getAvailableUrls());
+    } else {
+      urls.push(...server.getAvailableUrls().slice(0, options.numberOfFiles));
+    }
```

Comment:
yargs should always default this to a number for us, WDYT about 

```suggestion
    const maxNumberOfUrls = options.numberOfFiles || Infinity;
    urls.push(...server.getAvailableUrls().slice(0, maxNumberOfUrls));
```

Reply:
Oh cool I didn't know you could just pass Infinity to slice and make it work this way.
That's a lot cleaner than my if else block. Thanks.